ftest.sh

```diff
@@ -119,6 +120,15 @@ if [ \"\${HOSTNAME%%%%.*}\" != \"${nodes[0]}\" ]; then
             sudo mkdir -p /mnt/daos
         fi
     fi
+
+    # make sure to set up for daos_agent
+    sudo mkdir /var/run/daos_agent
+    sudo mkdir /var/run/daos_server
+    sudo chown ${current_host} -R /var/run/daos_agent
```

Comment:
And then here you want to escape `${}` so that the variable is expanded on the remote machine, not the one running the `pdsh` command:
```suggestion
    sudo chown \${current_username} -R /var/run/daos_agent
```